modular_skyrat/modules/events/code/modules/events/apc_failure.dm

```diff
@@ -0,0 +1,22 @@
+/datum/round_event_control/apc_failure
+	name = "APC Failure"
+	typepath = /datum/round_event/apc_failure
+	weight = 50
```

Comment:
This is far too high, match this to similar style events such as space dust. 20 I think.